includes/core/server/request/class-get-setup-intention.php

```diff
@@ -16,6 +16,13 @@
  * Request class for getting setup intents.
  */
 class Get_Setup_Intention extends Request {
+
+	/**
+	 * Used to set WordPress filter hook that will be executed when send() function is called.
+	 *
+	 * @var string $hook
+	 */
+	protected $hook = 'wcpay_get_setup_intent_request';
```

Comment:
There is an empty line before this declaration, which should be after it.

Reply:
Addressed [here](https://github.com/Automattic/woocommerce-payments/commit/e39ac1fe049de9ff790af034762fead702a8fbe9#diff-b9bbe01f6a7ecad3690036ffe391eebd60297fe950c118f8d2916dd4f3c1f345R26).